samples/highcharts/chart/events-fullscreenopen-fullscreenclose/demo.js

```diff
@@ -0,0 +1,43 @@
+Highcharts.chart('container', {
+
+    chart: {
+        events: {
+            beforeFullscreenOpen: function () {
+                this.update({
+                    title: {
+                        style: {
+                            fontSize: '30px'
+                        }
+                    },
+                    subtitle: {
+                        style: {
+                            color: 'red'
+                        }
+                    }
+                });
+            },
+            beforeFullscreenClose: function () {
+                this.update({
+                    title: {
+                        style: {
+                            fontSize: '18px'
+                        }
+                    }
+                });
+            }
+        }
+    },
+
+    title: {
+        text: 'Fullscreen'
+    },
+
+    subtitle: {
+        text: 'Click the context menu and choose "Fullscreen".<br> The title size will be larger'
```

Comment:
The button's text is "View in full screen", not "Fullscreen".

Also, there's unnecessary space after `<br>`.

Also, a dot is missing at the end of the sentence